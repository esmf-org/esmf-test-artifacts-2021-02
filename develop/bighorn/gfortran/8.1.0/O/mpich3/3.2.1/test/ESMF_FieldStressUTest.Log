20201222 132627.825 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201222 132627.825 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201222 132627.825 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201222 132627.825 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201222 132627.825 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201222 132627.825 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201222 132627.825 INFO             PET0 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_39-31-gaad96c2245
20201222 132627.825 INFO             PET0 ESMF library build date/time: "Dec 22 2020" "13:05:16"
20201222 132627.825 INFO             PET0 ESMF library build location : /scratch/esmf/esmf_test/cronjobs/trunk/gfortran/esmf
20201222 132627.825 INFO             PET0 ESMF_COMM                   : mpich3
20201222 132627.826 INFO             PET0 ESMF_NETCDF                 : disabled
20201222 132627.826 INFO             PET0 ESMF_PNETCDF                : disabled
20201222 132627.826 INFO             PET0 ESMF_PIO                    : enabled
20201222 132627.826 INFO             PET0 ESMF_YAMLCPP                : enabled
20201222 132627.826 INFO             PET0 ESMF_MOAB                   : enabled
20201222 132627.826 INFO             PET0  Beginning Test, file ESMF_FieldStressUTest.F90, line 51
20201222 132627.826 INFO             PET0  NUMBER_OF_PROCESSORS           1
20201222 132627.830 INFO             PET0  PASS  Create a simple Field, repeat 10 times, ESMF_FieldStressUTest.F90, line 61
20201222 132627.866 INFO             PET0  PASS  Create a simple Field, repeat 100 times, ESMF_FieldStressUTest.F90, line 69
20201222 132628.224 INFO             PET0  PASS  Create a simple Field, repeat 1000 times, ESMF_FieldStressUTest.F90, line 77
20201222 132632.216 INFO             PET0  PASS  Create a simple Field, repeat 10000 times, ESMF_FieldStressUTest.F90, line 85
20201222 132632.217 INFO             PET0  Ending Test, file ESMF_FieldStressUTest.F90, line 88
20201222 132632.217 INFO             PET0 Finalizing ESMF
Tue Dec 22 14:53:38 MST 2020
