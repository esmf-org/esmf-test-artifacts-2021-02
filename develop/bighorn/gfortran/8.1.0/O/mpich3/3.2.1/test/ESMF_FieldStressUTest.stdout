  Beginning Test, file ESMF_FieldStressUTest.F90, line 51
  NUMBER_OF_PROCESSORS           1
  PASS  Create a simple Field, repeat 10 times, ESMF_FieldStressUTest.F90, line 61
  PASS  Create a simple Field, repeat 100 times, ESMF_FieldStressUTest.F90, line 69
  PASS  Create a simple Field, repeat 1000 times, ESMF_FieldStressUTest.F90, line 77
  PASS  Create a simple Field, repeat 10000 times, ESMF_FieldStressUTest.F90, line 85
  Ending Test, file ESMF_FieldStressUTest.F90, line 88
  PET           0  Test Elapsed Time    4356.4459999999999       msec.
Thu Nov  5 05:30:26 MST 2020
